js/react/bundle.jsx

```diff
@@ -35,7 +35,7 @@ window.AgencyRegister = AgencyRegister;
 import Contact from './pages/contact.jsx';
 window.Contact = Contact;
 
-import DonationHistory from './pages/profile/donationhistory.jsx';
+import DonationHistory from './pages/profile/donation.jsx';
```

Comment:
why did you rename it? was `donationhistory` not correct?

Reply:
@Enubia yeah but it was not camel case haha so I wanted to match the current naming conventions

Reply:
oh but pages are fine with all lowercase :D

component file names should be PascalCase